starknet_py/common.py

```diff
@@ -40,3 +66,17 @@ def int_from_bytes(
     Converts the given bytes object (parsed according to the given byte order) to an integer.
     """
     return int.from_bytes(value, byteorder=byte_order, signed=signed)
+
+
+def get_storage_var_address(var_name: str, *args) -> int:
```

Comment:
Do we actually have any usecases of `args` in starknet.py? I know that cairo-lang's implementation had it but from my experience, I've never used it.

Reply:
As do we use `*args` in the code? We've got it in some places.

Reply:
I though we didn't. Let's keep it as is then.